src/hooks/useFarmStats.ts

```diff
@@ -1,19 +1,19 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useState } from 'react';
 
-import { BigIntish } from './types'
+import { BigIntish, TimeRemaining } from 'util/types';
 
-import gql from 'graphql-tag'
-import { Client } from 'urql'
+import dayjs from 'dayjs';
+import gql from 'graphql-tag';
+import { Client } from 'urql';
 
-import { WPOKT_SUBGRAPH_URL } from 'util/constants'
+import { getTimeRemaining } from 'util/helpers';
+import { WPOKT_SUBGRAPH_URL } from 'util/constants';
 
-const RETRY_EVERY = 3000
-const DAYS_IN_ONE_MONTH = 30
-const ONE_DAY = 86400000
-const ZERO_BIG_INT = BigInt(0)
+const RETRY_EVERY = 3000;
+const DAYS_IN_ONE_MONTH = 30;
+const ZERO_BIG_INT = BigInt(0);
```

Comment:
Bit of a nitpick, but:
```suggestion
const ZERO_BIG_INT = 0n;
```

Also, do we really need the `_BIG_INT` prefix? I believe that with typescript we'll realize if we're doing math with different types, so I think we can remove it. What do you think?